src/dashboard/src/components/api/urls.py

```diff
@@ -42,4 +42,7 @@
     url(r'administration/dips/atom/fetch_levels/$', views.fetch_levels_of_description_from_atom, name='fetch_atom_lods'),
     url(r'filesystem/metadata/$', views.path_metadata),
     url(r'processing-configuration/(?P<name>\w{1,16})', views.processing_configuration, name='processing_configuration'),
+    url(r'processing-configuration/(?P<name>\w{1,16})', views.processing_configuration),
```

Comment:
Is this OK having 2 URLs the same?

Reply:
Good catch, that was a conflict not solved properly, fixed now.